configuration_controller/tests/unit/fixtures/fake_mappings/request_mapping.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Same as above with YAML naming

Reply:
I'll create PR to change files from yaml to yml.